Specification/BrAPI-Schema/BrAPI-Germplasm/PedigreeNode.json

```diff
@@ -158,9 +158,9 @@
                 "primaryModel": false
             }
         },
-        "Progeny": {
+        "GermplasmChild": {
```

Comment:
@BrapiCoordinatorSelby this was proposed because 'Progeny' is plural.

Reply:
Should I change anything or leave it for now?